examples/asr/aishell/rnnlm.json

```diff
@@ -0,0 +1,46 @@
+{
+  "batch_size":32,
+  "num_epochs":100,
+  "sorta_epoch":0,
+  "ckpt":"examples/asr/hkust/ckpts/rnnlm",
+
+  "solver_gpu":[0],
+  "solver_config":{
+    "clip_norm":100,
+    "log_interval":10,
+    "enable_tf_function":true
+  },
+
+
+  "model":"rnnlm",
+  "num_classes": null,
+  "pretrained_model": null,
+  "model_config":{
+    "d_model": 512,
+    "rnn_type": "lstm",
+    "num_layer": 4,
```

Comment:
we still need to test the performance of 4 layer rnnlm verses 2 layer rnnlm, but it's ok for now